rust/src/bin/test/MyGame/Example/Monster.rs

```diff
@@ -0,0 +1,534 @@
+//! Automatically generated, do not modify.
```

Comment:
All these test files are also in tests/
They should probably not be also under rust/ if possible.